components/atoms/EChartWrapper/echart-wrapper.tsx

```diff
@@ -1,25 +1,27 @@
 import React from "react";
 import ReactECharts from "echarts-for-react";
 
-/* 
+/*
 Docs for eChart: https://echarts.apache.org/en/feature.html
 eChart Examples:  https://echarts.apache.org/examples/en/editor.html?c=scatter-simple&lang=ts
 Docs for ReactECharts: https://github.com/hustcc/echarts-for-react
  */
 
 interface EChartWrapperProps {
     option: Object;
+    className?: string;
 }
 
-const EChartWrapper: React.FC<EChartWrapperProps> = ({ option }) => {
+const EChartWrapper: React.FC<EChartWrapperProps> = ({ option, className }) => {
   return (
     <ReactECharts
       option={option}
+      className={className || ""}
```

Comment:
This was not referenced in your PR description. Can share more about your approach?

Reply:
Yeah, the [echarts-for-react](https://github.com/hustcc/echarts-for-react) component has a height of 300px and an absolutely positioned `canvas` that covers the pr table row. That's why it wasn't clickable.
| | |
| --- | --- |
| ![pr-table-row-1](https://user-images.githubusercontent.com/79809121/214910134-0e2ca0d4-708c-41b8-9111-8ba1ecf52498.png) | ![pr-table-row-2](https://user-images.githubusercontent.com/79809121/214910180-b10080c2-6d7c-4006-a8d0-b63f9104792f.png) |